media-source/idlharness.any.js

```diff
@@ -1,59 +0,0 @@
-// META: script=/resources/WebIDLParser.js
-// META: script=/resources/idlharness.js
-
-// https://w3c.github.io/media-source/
-
-'use strict';
-
-var mediaSource;
-var sourceBuffer;
-var video = document.createElement("video");
-
-promise_test(async t => {
-  const srcs = ['media-source', 'dom', 'html', 'url'];
-  const [idl, dom, html, url] = await Promise.all(
-      srcs.map(i => fetch(`/interfaces/${i}.idl`).then(r => r.text())));
-
-  var idlArray = new IdlArray();
-  idlArray.add_idls(idl);
-  idlArray.add_dependency_idls(dom);
-  idlArray.add_dependency_idls(html);
-  idlArray.add_dependency_idls(url);
-
-  const testIdls = new Promise(resolve => {
-    try {
-      mediaSource = new MediaSource();
-      video.src = URL.createObjectURL(mediaSource);
-      mediaSource.addEventListener("sourceopen", function () {
-        var defaultType ='video/webm;codecs="vp8,vorbis"';
-        if (MediaSource.isTypeSupported(defaultType)) {
-          sourceBuffer = mediaSource.addSourceBuffer(defaultType);
-        } else {
-          sourceBuffer = mediaSource.addSourceBuffer('video/mp4');
-        }
-        sourceBuffer.addEventListener("updateend", function (e) {
```

Comment:
@lukebjerring waiting for the updateend event looked unnecessary, so I didn't carry that over. Can you double check?